srcpkgs/asahi-firmware/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'asahi-firmware'
+pkgname=asahi-firmware
+version=0.7.6
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 lzfse"
+short_desc="Asahi Linux firmware tools"
+maintainer="dkwo <npiazza@disroot.org>"
+license="MIT"
+homepage="https://github.com/AsahiLinux/asahi-installer"
+distfiles="https://github.com/AsahiLinux/asahi-installer/archive/refs/tags/v${version}.tar.gz"
+checksum=2a8832fa206272ecdd1111b09b6b68010bf1b1adccc57a1eb98ba2213ab8b440
+
```

Comment:
```suggestion
make_dirs="/usr/lib/firmware/vendor 0755 root root"
```
this directory must exist for the dracut hook to successfully mount the firmware

Reply:
I see. asahi-scripts is creating it, but void's scripts are removing it as an empty dir. let me add it there.